### ISSUE TITLE
Debug stepping inconsistencies for getters

### ISSUE THREAD

**Issue by jensjoha** [dart-lang/sdk#29067]
E.g.

```
get fooXYZ => "fooXYZ";

get fooXYZ2 {
  int i = 42;
  return "Hello, $i!";
}
```

(i.e. top-level) will stop (when stepping into the getter) first at column 12 (in fooXYZ, i.e. on '=>') and 15 (in fooXYZ2, i.e. on '{').

```
class Bar {
  get barXYZ => "barXYZ";

  get barXYZ2 {
    int i = 42;
    return "Hello, $i!";
  }
}
```

(i.e. getter in a class) will instead stop (when stepping into the getter) at column 3 (i.e. on 'get') in both cases.

I.e. for getters the debugger stops either on 'get' or on whatever is after the name depending on whether it's in a class or not.

**Comment by mhausner** [comment]
If the getter in class Bar is made static, it behaves the same way as the top-level getter. So, the difference is due to static vs instance getter.

**Comment by mhausner** [comment]
https://codereview.chromium.org/2751523009/

**Comment by mhausner** [comment]
Fixed in e78b828d101c24f7e2f0c005a38a2b0555d879a7.
